src/hyperfine/benchmark.rs

```diff
@@ -86,7 +96,12 @@ pub fn mean_shell_spawning_time(style: &OutputStyleOption) -> io::Result<TimingR
 
     for _ in 0..COUNT {
         // Just run the shell without any command
-        let res = time_shell_command(&Command::new(""), CmdFailureAction::RaiseError, None);
+        let res = time_shell_command(
+            &Command::new(""),
+            print_stdout,
```

Comment:
Do we really want to show the output of the empty shell command or just pass `false` here?

Reply:
I wasn't too sure about that. From the name alone I thought that `time_shell_command` would be used to deduct the shell startup time from the actual application runtime, so I wanted to keep it as similar to the actual benchmark as possible.

If that's not the case or the time difference is irrelevant (it should be pretty much 0), then I can just remove it.

Reply:
Yeah okay, I guess that makes sense. Let's leave it in there.